art-common/pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
```

Comment:
Would love to have this project definition one level up. Or what is the reason to declare this an independent package?

Reply:
I think having an independent package is better. For things like art-bot and maybe ocp-build-data-validator, they probably don't need the code and dependencies in elliott or doozer. If we keep everything in a single package, that means we have to install all python code and dependencies together.

Also there is another good reason for me: Elliott works on macOS but Doozer doesn't. I have elliott installed locally on my Mac but if we make everything as a single package, I won't be able to elliott natively :(

Reply:
working on it at https://github.com/openshift-eng/art-tools/pull/96 (includes this)

Reply:
closed #96; refactoring into a single project may be worthwhile but yuxiang makes good points and i think it's too big a change for this PR.